src/app/config.js

```diff
@@ -0,0 +1,339 @@
+                        name: 'REHABILITATION',
+                        queryValue: '013' },
+                    {
+                        name: 'CRITICAL ACCESS',
+                        queryValue: '014' },
+                    {
+                        name: 'LONG TERM CARE',
+                        queryValue: '015' },
+                    {
+                        name: 'CHILDRENS',
+                        queryValue: '016' },
+                    {
+                        name: 'CHEMICAL DEPENDENCY',
+                        queryValue: '01A' }]
+            },
+            {
+                name: 'NURSING HOME',
+                queryValue: 'NURSING HOME',
+                children: [
+                    {
+                        name: 'SMALL - PRIVATE PAY (2 - 3 RESIDENTS)',
+                        queryValue: 'S11' },
+                    {
+                        name: 'NURSING',
+                        queryValue: '024' },
+                    {
+                        name: 'SKILLED / NURSING - MEDICARE/MEDICAID',
+                        queryValue: '021' },
+                    {
+                        name: 'SKILLED NURSING - MEDICARE',
+                        queryValue: '022' },
+                    {
+                        name: 'SKILLED NURSING',
+                        queryValue: '023' },
+                    {
+                        name: 'NURSING HOME - PRIVATE PAY',
+                        queryValue: '02C' },
+                    {
+                        name: 'INTELLECTUALLY DISABLED INTERMEDIATE CARE',
+                        queryValue: '111' }]
+            },
+            {
+                name: 'BIRTHING CENTER',
+                queryValue: 'S41',
+                children: []
+            },
+            {
+                name: 'PSYCHIATRIC RESIDENTIAL TREATMENT',
+                queryValue: '061',
+                children: []
+            },
+            {
+                name: 'PORTABLE X-RAY SUPPLIERS',
+                queryValue: '071',
+                children: []
+            },
+            {
+                name: 'PHYSICAL THERAPY/SPEECH PATHOLOGY',
+                queryValue: '081',
+                children: []
+            },
+            {
+                name: 'RURAL HEALTH CLINICS',
+                queryValue: '121',
+                children: []
+            },
+            {
+                name: 'COMPREHENSIVE OUTPATIENT REHABILITATION',
+                queryValue: '141',
+                children: []
+            },
+            {
+                name: 'ABORTION CLINIC',
+                queryValue: 'ABORTION CLINIC',
+                children: [
+                    {
+                        name: 'ABORTION CLINIC TYPE 1',
+                        queryValue: 'S51' },
+                    {
+                        name: 'ABORTION CLINIC TYPE 2',
+                        queryValue: 'S52' }]
+            }
+        ],
+        cities: ['ALPINE',
```

Comment:
This could be gathered from the web api. `agrc/modules/webapi` makes it pretty easy. But maybe you are wanting to save that hit on the server each time the app is loaded?

e.g. Millcreek isn't in this list.